src/platform/Ameba/ConfigurationManagerImpl.cpp

```diff
@@ -79,6 +79,21 @@ CHIP_ERROR ConfigurationManagerImpl::Init()
         SuccessOrExit(err);
     }
 
+    if (!AmebaConfig::ConfigValueExists(AmebaConfig::kConfigKey_RegulatoryLocation))
+    {
+        uint32_t regulatoryLocation = to_underlying(chip::app::Clusters::GeneralCommissioning::RegulatoryLocationType::kIndoor);
+        err                         = WriteConfigValue(AmebaConfig::kConfigKey_RegulatoryLocation, regulatoryLocation);
+        SuccessOrExit(err);
+    }
+
+    if (!AmebaConfig::ConfigValueExists(AmebaConfig::kConfigKey_LocationCapability))
+    {
+        uint32_t locationCapability =
+            to_underlying(chip::app::Clusters::GeneralCommissioning::RegulatoryLocationType::kIndoorOutdoor);
```

Comment:
```suggestion
            to_underlying(app::Clusters::GeneralCommissioning::RegulatoryLocationType::kIndoorOutdoor);
```

Reply:
That said, is "indoor+outdoor" a safe default?  My understanding is that anything that claims "outdoor" is subject to more stringent regulation in some countries...

Reply:
As a platform provider, we try to support as many features as possible. We leave it to our customers' discretion whether they want to support outdoors or not.

Reply:
Sure, but here we're talking about default values if the customer does not indicate anything otherwise, right?